maintainer/maintainer/ethereum/shared.py

```diff
@@ -32,13 +29,16 @@ def _nonce(i: int) -> Iterator[int]:
 async def init() -> None:
     '''Set up a connection to the interwebs'''
     global CONNECTION
+    global MAX_GAS_PRICE
```

Comment:
I've noticed that configurable variables are generally read from config wherever they are needed and not stored globally in Relay Maintainer, e.g:
URL = config.get()['BCOIN_URL']

Perhaps we could limit the number of global variables by doing the same with MAX_GAS_PRICE?
Just a suggestion.